### PR TITLE
Add pip package

### PR DESCRIPTION
**Pull request by GlennWSo**
Made an install-able package
and set the cli tool as an entry point for using the package
Resolves issue #2 

- moved source code and schemas
- created an install-able package
- ignore typical python packaging stuff used template: https://raw.githubusercontent.com/github/gitignore/main/Python.gitignore
- import from package instead of filesystem relative
- added hint pip install hint for devs
- Moved cli main body to a function
- added entry-point to the package for using the package as cli tool

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+from setuptools import setup
+
+setup(
```
Comment (mloskot) [inline setup.py:3]: You could have added header comment and credit yourself.

**Review by mloskot — APPROVED** [review]
@GlennWSo  Awesome! Thank you for your contribution.

If you could also work out pushing the package that would be very much appreciated. If you need commit access to this repo, I see no problem to grant you with it.

**Comment by GlennWSo** [comment]
@mloskot 
>If you need commit access to this repo, I see no problem to grant you with it.

I think that would be helpful, i would like to able to create tags and releases in this repo to show what was used for the pypi releases.

**Comment by mloskot** [comment]
@GlennWSo  Invite sent

**Pull request merged**